gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -133,7 +146,9 @@ private void addNewObservation(Feature feature, Form form) {
     navigator.navigate(HomeScreenFragmentDirections.addObservation(projectId, featureId, formId));
   }
 
-  /** This is only possible after updating the location of the feature. So, reset the UI. */
+  /**
+   * This is only possible after updating the location of the feature. So, reset the UI.
```

Comment:
Please add brief method description before this clarification (i.e., what is "this" in this case?). Also consider rewording the second phrase into a complete sentence.